examples/adwords_axis/src/main/java/adwords/axis/v201802/accountmanagement/GetAccountChanges.java

```diff
@@ -0,0 +1,208 @@
+          DEFAULT_CONFIGURATION_FILENAME, ve);
+      return;
+    } catch (OAuthException oe) {
+      System.err.printf(
+          "Failed to create OAuth credentials. Check OAuth settings in the %s file. "
+              + "Exception: %s%n",
+          DEFAULT_CONFIGURATION_FILENAME, oe);
+      return;
+    }
+
+    AdWordsServicesInterface adWordsServices = AdWordsServices.getInstance();
+
+    try {
+      runExample(adWordsServices, session);
+    } catch (ApiException apiException) {
+      // ApiException is the base class for most exceptions thrown by an API request. Instances
+      // of this exception have a message and a collection of ApiErrors that indicate the
+      // type and underlying cause of the exception. Every exception object in the adwords.axis
+      // packages will return a meaningful value from toString
+      //
+      // ApiException extends RemoteException, so this catch block must appear before the
+      // catch block for RemoteException.
+      System.err.println("Request failed due to ApiException. Underlying ApiErrors:");
+      if (apiException.getErrors() != null) {
+        int i = 0;
+        for (ApiError apiError : apiException.getErrors()) {
+          System.err.printf("  Error %d: %s%n", i++, apiError);
+        }
+      }
+    } catch (RemoteException re) {
+      System.err.printf(
+          "Request failed unexpectedly due to RemoteException: %s%n", re);
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param adWordsServices the services factory.
+   * @param session the session.
+   * @throws ApiException if the API request failed with one or more service errors.
+   * @throws RemoteException if the API request failed due to other errors.
+   */
+  public static void runExample(AdWordsServicesInterface adWordsServices, AdWordsSession session)
+      throws RemoteException {
+    // Get the CampaignService.
+    CampaignServiceInterface campaignService =
+        adWordsServices.get(session, CampaignServiceInterface.class);
+
+    // Get the CustomerSyncService.
+    CustomerSyncServiceInterface customerSyncService =
+        adWordsServices.get(session, CustomerSyncServiceInterface.class);
+
+    // Get a list of all campaign IDs.
+    List<Long> campaignIds = new ArrayList<Long>();
+    Selector selector = new SelectorBuilder()
+        .fields(CampaignField.Id)
+        .build();
+    CampaignPage campaigns = campaignService.get(selector);
+    if (campaigns.getEntries() != null) {
+      for (Campaign campaign : campaigns.getEntries()) {
+        campaignIds.add(campaign.getId());
+      }
+    }
+
+    // Create date time range for the past 24 hours.
+    DateTimeRange dateTimeRange = new DateTimeRange();
+    dateTimeRange.setMin(new SimpleDateFormat("yyyyMMdd HHmmss").format(new Date(System
+        .currentTimeMillis() - 1000L * 60 * 60 * 24)));
+    dateTimeRange.setMax(new SimpleDateFormat("yyyyMMdd HHmmss").format(new Date()));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.